vitest.config.ts

```diff
@@ -1,24 +1,53 @@
-import { defineConfig } from 'vitest/config';
+import { defaultExclude, defineConfig } from 'vitest/config';
 import tsconfigPaths from 'vite-tsconfig-paths';
 
 export const TEST_TIMEOUT = 50_000;
+const defaultInclude = '**/*.{test,spec}.?(c|m)[jt]s?(x)';
```

Comment:
I want to prepend this default include string with another path, so chose not to use `defaultInlude` from `vitest/config` as it's an array.